qa/workunits/rbd/concurrent.sh

```diff
@@ -88,9 +88,15 @@ function setup() {
 
 	# Set up some environment for normal teuthology test setup.
 	# This really should not be necessary but I found it was.
-	TOP="/tmp/cephtest"
+
+        if [ -f /etc/ceph/ceph.conf ]; then
+            TOP="/etc/ceph"
+            export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/ceph.client.0.keyring"
+        else
+            TOP="/tmp/cephtest"
+            export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/data/client.0.keyring"
+        fi
 	export CEPH_ARGS="--conf ${TOP}/ceph.conf"
-	export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/data/client.0.keyring"
 	export CEPH_ARGS="${CEPH_ARGS} --name client.0"
```

Comment:
nice catch, this is all from before teuthology installed packages. These settings are wrong now, and most can be removed since the default should be fine. We didn't notice this failing since it doesn't use set -e.

I think the only environment variable that we want to keep is adding --name client.0 to CEPH_ARGS so that we used that user instead of client.admin.